conf.py

```diff
@@ -51,8 +52,8 @@
 
 # General information about the project.
 project = u'Open edX Proposals'
-copyright = u'2016, Calen Pennington'
-author = u'Calen Pennington'
+copyright = u'2016, edX Inc. and licensed under a Creative Commons Attribution-ShareAlike 4.0 International License unless otherwise specified'
```

Comment:
Is this what PEP's do?  One copyright notice to cover all the documents doesn't seem right: the date will need to be different for different ones.